.env.example

```
@@ -1,8 +1,7 @@
-# [required] 1password
-OP_VAULT=xxxx # uuid
-OP_ITEM=xxxx  # uuid
-OP_TOTP_FIELD=xxxx # uuid
-OP_SERVICE_ACCOUNT_TOKEN=
+# [required] MoneyForward login
+MONEY_FORWARD_EMAIL=user-a@example.com
+MONEY_FORWARD_PASSWORD=
+MONEY_FORWARD_TOTP_SECRET= # base32 secret or otpauth:// URI
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)